content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -61,7 +61,7 @@ The following bulk operations run as transactions:
 
 > Refer to the 📖 [transactions guide](../../../guides/performance-and-optimization/prisma-client-transactions-guide#bulk-operations) for more examples.
 
-## The `transaction` API
+## The `$transaction` API
```

Comment:
To make the inline code show as inline code in the sidenavs we need to use the `<inlinecode>` element. 😊

`## The <inlinecode>$transaction</inlinecode> API`

Reply:
```suggestion
## The <inlinecode>$transaction</inlinecode> API
```